workflows/scflow.nf

```diff
@@ -27,7 +27,7 @@ if (params.celltype_mappings) { ch_celltype_mappings = file(params.celltype_mapp
 if (params.ensembl_mappings) { ch_ensembl_mappings = file(params.ensembl_mappings, checkIfExists: false) }
 if (params.ensembl_mappings) { ch_ensembl_mappings2 = file(params.ensembl_mappings, checkIfExists: false) }
 if (params.ensembl_mappings) { ch_ensembl_mappings3 = file(params.ensembl_mappings, checkIfExists: false) }
-if (params.reddim_genes_yml) { ch_reddim_genes_yml = file(params.reddim_genes_yml, checkIfExists: true) }
+if (params.reddim_genes_yml) { ch_reddim_genes_yml = file(params.reddim_genes_yml, checkIfExists: false) }
```

Comment:
The logic is intended to check for file existence only where the file is specified. Are you seeing it run this check even without the file being provided?

Reply:
This is probably the issue in fact: https://github.com/nf-core/scflow/blob/c49202faf0100a630cc63568e6e31521726d51ec/nextflow.config#L34 (i.e. there's a default value for this 'optional' input).

Reply:
Yes I try to not specify anything i.e. remove the value that was previously set, but the pipeline still expects it because of the line above

Reply:
What I'm saying is, the `if (params.reddim_genes_yml) {` should prevent the existence check from happening where the parameter is not supplied, which implies that the problem is the value is getting set inappropriately somewhere, rather than the existence check itself.